autonomous/forward.py

```diff
@@ -8,7 +8,10 @@ class Forward(AutonomousStateMachine):
 
     drivetrain = Drivetrain
 
-    @state(first=True)
+    @state(first=True)                         '''This tells us this state is the starting state and that it is not timed.'''
```

Comment:
The `@state` decorator and `@timed_state` decorator are not decorators that we wrote. They are a part of magicbot, which already has really good documentation. Rather than including all there documentation in our code, why not use their documentation? That's what it's for.

http://robotpy-wpilib-utilities.readthedocs.io/en/latest/robotpy_ext.autonomous.html#robotpy_ext.autonomous.stateful_autonomous.state